common/messages/messages_en.go

```diff
@@ -163,6 +163,13 @@ var messagesEn = map[string]util.Message{
 		RC:          400,
 		Action:      "Review available volume profiles and IOPS in the IBM Cloud Block Storage for VPC documentation https://cloud.ibm.com/docs/vpc-on-classic-block-storage?topic=vpc-on-classic-block-storage-block-storage-profiles.",
 	},
+	"VolumeProfileEmpty": {
+		Code:        "VolumeProfileEmpty",
+		Description: "Empty information could not be found.",
```

Comment:
can you update this description may be following 

Volume profile is empty, you need to pass valid profile name

Reply:
Updated